src/Aquarius.Client/TimeSeries/Client/AquariusClient.cs

```diff
@@ -327,5 +335,20 @@ private void ClearConnectedClients<TKey>(Dictionary<TKey, IServiceClient> client
 
             clientDictionary.Clear();
         }
+
+        public void UpdateBearerToken(string accessToken)
```

Comment:
Not a deal breaker. How about naming this method `UpdateAccessToken` at this level of abstraction.  Only the `ClientHelper` knows about and encapsulates that the access token is actually set as a bearer token in the auth header.

Reply:
Good suggestion, done